x/concentrated-liquidity/lp_test.go

```diff
@@ -82,6 +84,17 @@ func (s *KeeperTestSuite) TestCreatePosition() {
 			amount1Desired:    sdk.ZeroInt(),
 			expectedError:     errors.New("liquidityDelta calculated equals zero"),
 		},
+		"create a position with non default tick spacing (10) with ticks that fall into tick spacing requirements": {
+			lowerTick:       int64(84220),
+			upperTick:       int64(86130),
+			amount0Expected: sdk.NewInt(997568),
+			liquidityAmount: sdk.MustNewDecFromStr("1514719247.706987476085061790"),
+			tickSpacing:     10,
+		},
+		"error: attempt to create a position that does not fall into tick spacing requirement of that pool": {
```

Comment:
Can we add a comment that default lower tick and default upper tick are not divisible by the tick spacing of 10. Wasn't immediately apparent to me why this would error.